modules/event-exporter/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "chart_version" {
+  type        = string
+  default     = ""
+  description = "The app chart version to use"
+}
+
+variable "namespace" {
+  description = "The namespace to install app helm."
+  type        = string
+  default     = "meta-system"
+}
+
+variable "create_namespace" {
+  type        = bool
+  default     = false
+  description = "Create namespace if requested"
+}
+
+variable "helm_values" {
```

Comment:
there should be some required configurations, like webhook endpoint which we have to specify in values, can be here or as separate variable. This is needed for module users to know how to use and pass needed variables,
also please have tests/examples folder in this submodule with basic use case